ReactWindows/ReactNative.Net46/ReactPage.cs

```diff
@@ -100,8 +101,18 @@ public virtual string JavaScriptMainModuleName
         /// <param name="arguments">The launch arguments.</param>
         public void OnCreate(string[] arguments)
         {
+            OnCreate(arguments, default(JObject));
+        }
+
+        /// <summary>
+        /// Called when the application is first initialized.
+        /// </summary>
+        /// <param name="arguments">The launch arguments.</param>
+        /// <param name="initialProps">The initialProps.</param>
+        public void OnCreate(string[] arguments, JObject initialProps)
+        {
             ApplyArguments(arguments);
-            RootView.StartReactApplication(ReactInstanceManager, MainComponentName);
+            RootView.StartReactApplication(ReactInstanceManager, MainComponentName, default(JObject));
```

Comment:
errr, shouldn't we be passing the supplied initialProps argument down into the RootView?